applications/common/server/common/__main__.py

```diff
@@ -1,23 +1,45 @@
+#!/usr/bin/env python3
+
 import os
+import logging
 
-from flask import Flask
 from flask_cors import CORS
 
 import connexion
 
 from common import encoder
 
-def main():
-    app = connexion.App(__name__, specification_dir='./openapi/')
-    app.app.config.from_object(os.environ['APP_SETTINGS'])
-    app.app.json_encoder = encoder.JSONEncoder
-    app.add_api('openapi.yaml',
-                arguments={'title': 'CH service API'},
-                pythonic_params=True)
+connexion_app = connexion.App(__name__, specification_dir='./openapi/')
```

Comment:
please wrap the whole initialisation into a CH function, looks to me that this is everytime the same

please use cloudharness.log and remove the unicorn logger stuff (2 weeks ago I formatted the default CH logger)